scripts/start.sh

```diff
@@ -67,6 +67,21 @@ if [ -n "${RCON_PORT}" ]; then
     sed -i "s/RCONPort=[0-9]*/RCONPort=$RCON_PORT/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini
 fi
 
+if [[ -n "${BACKUP_ENABLED}" ]]; then
+    echo "BACKUP_ENABLED=${BACKUP_ENABLED}"
+
+    if [[ -z "${BACKUP_CRON_EXPRESSION}" ]]; then
+        printf "\e[0;31m***** BACKUP_CRON_EXPRESSION NOT SET *****\e[0m\n"
+        exit 1
+    else
+        echo "BACKUP_CRON_EXPRESSION=${BACKUP_CRON_EXPRESSION}"
+    fi
+
+    echo "${BACKUP_CRON_EXPRESSION} root bash /usr/local/bin/backup" > /etc/cron.d/backups-cron
```

Comment:
this needs to be in the "else" clause, this will fail if BACKUP_CRON_EXPRESSION is empty